content/docs/search/rules.md

```diff
@@ -91,51 +91,97 @@ Here are some example conditions:
 
 ### Replace Search Term
 
-Helps in replacing the user’s entire search query with another query. Helps in showing relevant results to users, specially when you are aware from the analytics that certain search term is returning no results.
+It helps in replacing the user’s entire search query with another query. Helps in showing relevant results to users, especially when you are aware of the analytics that certain search term is returning no results.
 
 ![replace search term](https://www.dropbox.com/s/p0he4889pkbl1u8/Screenshot%202020-02-20%2010.50.10.png?raw=1)
 
+### Replace Search Query
+This action gives you more control over the search query in a way that you can modify the whole query by using the [ElasticSearch's Query String](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html) syntax.
+
+For example, the following query returns all the documents for which  `height` and `width` matches `10` and `20` respectively.
+
+```bash
+height:10 + width:20
+```
+The above is just an example of ElasticSearch query string syntax where values are fixed. You can also use the variable values captured from the `query` value as per the regular expression defined in the trigger expression.
+
+For example, let's define the trigger expression with the help of the `advanced editor`.
+
+![trigger expression](https://i.imgur.com/mgNERHg.png)
+
+The above rule will only get applied when the query value is of the following pattern.
+
+```bash
+{number_x} x {number_y}
+```
+
+Now define the `query` value which will replace the original query.
+
+```bash
+height:$1 + width:$2
+```
+
+Here `$1` and `$2` are variables that will get extracted from the regex capture groups defined in the trigger expression. Now if some user makes a search with value as `10 x 20` then the final query will be `height:10 + width:20`.
+
+![replace search query](https://i.imgur.com/6kjzXz6.png)
+
+
 ### Remove Words
 
 Removing words is the progressive loosening of query constraints to include more results when none are initially found.
 
 For example, imagine an online smartphone shop that sold a limited inventory of iPhones in only 16GB and 32GB varieties. Users searching for “iphone 5 64gb” would see no results. This is not ideal behavior - it would be far better to show users some iPhone 5 results instead of a blank page.
 
-You can remove multiple words by space separated values. E.g. `iphone samsung`.
+You can remove multiple words by space-separated values. E.g. `iphone samsung`.
 
-![remove words](https://i.imgur.com/ArDcJRn.png)
+![remove words](https://i.imgur.com/bVM7oFg.png)
 
 ### Replace Words
 
 Rules offers an alternative. You can now replace words instead of adding new ones. For example, if you make `tv` a synonym for `television`, Rules will replace `tv` with `television` so that only `television` is used to search.
 
 ![replace-word](https://i.imgur.com/ps6JH9A.png)
```

Comment:
@bietkul This needs a new image based on the latest UI.

Reply:
@siddharthlatest There is no change here.